t/inputs/import_csv_file/test.converted.csv

```diff
@@ -0,0 +1,2 @@
+code	producer_product_id	producer_version_id	lc	product_name_fr	abbreviated_product_name_fr	generic_name_fr	.2	.3	.4	.5	packaging	packaging_text_fr	brands	brand_owner	categories	labels	countries	stores	obsolete	obsolete_since_date	periods_after_opening	origins	manufacturing_places	producer_fr	emb_codes	ingredients_text_fr	allergens	traces	energy-kj_100g_value_in_kj	energy-kcal_100g_value_in_kcal	.6	.7	.8	.9	.10	.11	.12	.13	.14	nutriscore_score_producer	nutriscore_grade_producer	nova_group_producer	conservation_conditions_fr	warning_fr	preparation_fr	recipe_idea_fr	recycling_instructions_to_recycle_fr	recycling_instructions_to_discard_fr	customer_service_fr	link	image_front_url_fr	image_ingredients_url_fr	image_nutrition_url_fr	image_packaging_url_fr	image_other_url
```

Comment:
Aren't you creating this file in the test? (as well as test.csv)
If so then we should not need them in git as they won't be used.

Reply:
My bad, they should not have been commited (and I though I did not !)